2007-05-25  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/unix/sysv/linux/arm/nptl/sysdep-cancel.h
	(RTLD_SINGLE_THREAD_P): Define.

Index: b/ports/sysdeps/unix/sysv/linux/arm/nptl/sysdep-cancel.h
===================================================================
--- a/ports/sysdeps/unix/sysv/linux/arm/nptl/sysdep-cancel.h	2008-08-08 18:37:50.000000000 +0200
+++ b/ports/sysdeps/unix/sysv/linux/arm/nptl/sysdep-cancel.h	2008-08-08 18:37:51.000000000 +0200
@@ -138,3 +138,9 @@
   __builtin_expect (THREAD_GETMEM (THREAD_SELF, \
 				   header.multiple_threads) == 0, 1)
 #endif
+
+#ifndef __ASSEMBLER__
+# define RTLD_SINGLE_THREAD_P \
+  __builtin_expect (THREAD_GETMEM (THREAD_SELF, \
+				   header.multiple_threads) == 0, 1)
+#endif
